tests/tests/resource_limiting.rs

```diff
@@ -159,6 +217,28 @@ async fn run_tests_on_ws_server(server_addr: SocketAddr, server_handle: WsServer
 	assert!(pass_mem.is_ok());
 	assert_server_busy(fail_mem);
 
+	// 3 CPU units (manually set for subscriptions) per call, the sink is dropped immediately
+	let (pass1, pass2, pass3) = tokio::join!(
+		client.subscribe::<i32>("subscribe_hello", None, "unsubscribe_hello"),
+		client.subscribe::<i32>("subscribe_hello", None, "unsubscribe_hello"),
+		client.subscribe::<i32>("subscribe_hello", None, "unsubscribe_hello"),
+	);
```

Comment:
Am I right in thinking that even if there were 10 client.subscribes here, they would all be OK because something is dropped immediately, and there is no race condition where that might not always be true?

Reply:
(if so, could the comment be tweaked to say something like "the sink is dropped immediately, so no resources are claimed and all of the calls succeed"?"

Reply:
I believe that to be the case for this particular test.

We submit all the `subscribe` requests from one client to the server. The server handles each client on a dedicated `background_task` task. That means all our submitted `subscribe` requests get handled serially, on the same background task.
The subscription handling would: acquire + update + release resource table, call subscription callback, drop the sink immediately, acquire + update + release resource table.

The resource limiting would reject connections for multiple clients if they attempt to make a subscription call to a method that drops the sink immediately.